src/current/v23.1/migrate-from-postgres.md

```diff
@@ -6,254 +6,291 @@ keywords: copy
---------------------+-----------+--------------------+--------+---------------+----------------+----------
- 381845110403104769 | succeeded |                  1 | 300024 |             0 |              0 | 11534293
-(1 row)
-~~~
-
-### Import a table from a full database dump
-
-This example assumes you [dumped the entire database](#dump-the-entire-database).
-
-[`IMPORT`][import] can import one table's data from a full database dump. It reads the data and applies any `CREATE TABLE` statements from the file.
-
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> CREATE DATABASE IF NOT EXISTS employees;
-> USE employees;
-> IMPORT TABLE employees FROM PGDUMP 'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/employees-full.sql.gz' WITH ignore_unsupported_statements;
-~~~
-
-~~~
-       job_id       |  status   | fraction_completed |  rows  | index_entries | system_records |  bytes
---------------------+-----------+--------------------+--------+---------------+----------------+----------
- 383839294913871873 | succeeded |                  1 | 300024 |             0 |              0 | 11534293
-(1 row)
-~~~
-
-### Import a table from a table dump
-
-The simplest way to import a table dump is to run [`IMPORT`][import]. It reads the table data and any `CREATE TABLE` statements from the file:
+### Step 1. Convert the PostgreSQL schema
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> CREATE DATABASE IF NOT EXISTS employees;
-> USE employees;
-> IMPORT PGDUMP 'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/employees.sql.gz' WITH ignore_unsupported_statements;
-~~~
+Use the [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) to convert the `frenchtowns` schema for compatibility with CockroachDB. The schema has three tables: `regions`, `departments`, and `towns`.
 
-~~~
-       job_id       |  status   | fraction_completed |  rows  | index_entries | system_records |  bytes
---------------------+-----------+--------------------+--------+---------------+----------------+----------
- 383855569817436161 | succeeded |                  1 | 300024 |             0 |              0 | 11534293
-(1 row)
-~~~
+1. Dump the PostgreSQL `frenchtowns` schema with the following [`pg_dump`](https://www.postgresql.org/docs/15/app-pgdump.html) command:
 
-## Configuration Options
+       {% include_cached copy-clipboard.html %}
+       ~~~ shell
+       pg_dump --schema-only frenchtowns > frenchtowns_schema.sql
+       ~~~
 
-The following options are available to `IMPORT PGDUMP`:
+1. Open the [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) in the {{ site.data.products.db }} Console and start the process to [add a new PostgreSQL schema](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page#convert-a-schema).
```

Comment:
Can we say "add a new PostgreSQL schema" instead of "start the process to add a new PostgreSQL schema"? The latter sounds confusing to me (it makes me wonder if I'm supposed to follow only part of the linked instructions).

Reply:
Done!